doc/rbd/iscsi-target-cli.rst

```diff
@@ -149,9 +149,14 @@ For rpm based instructions execute the following commands:
    ::
 
        # systemctl daemon-reload
+       
+       # systemctl enable rbd-target-gw
+       # systemctl enable rbd-target-gw
```

Comment:
```suggestion
       # systemctl start rbd-target-gw
```

Reply:
Nit: `rbd-target-gw` depends on `rbd-target-api`, so while not strictly required, it would be cleaner to show it getting enabled/started after `rbd-target-api`